libs/expo/shared/ui-components/src/lib/LibraryModal/index.tsx

```diff
@@ -0,0 +1,166 @@
+import { ReactNativeFile } from '@monorepo/expo/shared/apollo';
+import { Colors, Spacings } from '@monorepo/expo/shared/static';
+import { resizeImage } from '@monorepo/expo/shared/utils';
+import { useCameraPermissions } from 'expo-camera';
+import * as ImagePicker from 'expo-image-picker';
+import { useState } from 'react';
+import { Alert, Pressable, View } from 'react-native';
+import Modal from 'react-native-modal';
+import LibCamera from '../Camera';
```

Comment:
Is there a reason this needs to be aliased? Can't we just call it Camera the way we do in libs/expo/shared/ui-components/src/lib/InteractionCamera/index.tsx? If it does have to be aliased, do we want to alias it in both places? e.g., DocLibraryCamera and InteractionCamera

Reply:
similar comment.. can we rename `LibraryModal` to something more specific, like `ClientDocumentModal`. I feel like "library" is a very general word and can be confusing

Reply:
it is general component. not specific for client. 
we can use it everywhere. maybe MediaPickerModal ?

Reply:
`MediaPickerModal` sgtm

Reply:
ok